<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QwtPlot" name="Qwt_Widget" native="true">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>400</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QComboBox" name="UserCurve">
        <property name="maximumSize">
         <size>
          <width>221</width>
          <height>27</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="labelX">
          <property name="maximumSize">
           <size>
            <width>11</width>
            <height>27</height>
           </size>
          </property>
          <property name="text">
           <string>X</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditX">
          <property name="maximumSize">
           <size>
            <width>202</width>
            <height>27</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="labelY">
          <property name="maximumSize">
           <size>
            <width>9</width>
            <height>27</height>
           </size>
          </property>
          <property name="text">
           <string>Y</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditY">
          <property name="maximumSize">
           <size>
            <width>204</width>
            <height>27</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="add">
        <property name="maximumSize">
         <size>
          <width>219</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="free">
        <property name="maximumSize">
         <size>
          <width>219</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="TwoL">
          <property name="minimumSize">
           <size>
            <width>41</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>41</width>
            <height>27</height>
           </size>
          </property>
          <property name="text">
           <string>&lt;&lt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="OneL">
          <property name="minimumSize">
           <size>
            <width>41</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>41</width>
            <height>27</height>
           </size>
          </property>
          <property name="text">
           <string>&lt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="OneR">
          <property name="minimumSize">
           <size>
            <width>41</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>41</width>
            <height>27</height>
           </size>
          </property>
          <property name="text">
           <string>&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="TWoR">
          <property name="minimumSize">
           <size>
            <width>41</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>41</width>
            <height>27</height>
           </size>
          </property>
          <property name="text">
           <string>&gt;&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="Sort">
        <property name="text">
         <string>Sort current curve</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="SortIfCheched">
        <property name="text">
         <string>Sort with add point (all points)</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>710</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionOpen_file"/>
    <addaction name="actionSave_File"/>
    <addaction name="actionSave_File_as"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionNew_curve"/>
    <addaction name="actionDelete_curve"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_one_curve"/>
    <addaction name="actionSave_current_curve"/>
    <addaction name="actionSave_current_curve_as"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_file">
   <property name="text">
    <string>Open file</string>
   </property>
  </action>
  <action name="actionSave_File">
   <property name="text">
    <string>Save File</string>
   </property>
  </action>
  <action name="actionSave_File_as">
   <property name="text">
    <string>Save File as..</string>
   </property>
  </action>
  <action name="actionNew_curve">
   <property name="text">
    <string>New curve</string>
   </property>
  </action>
  <action name="actionDelete_curve">
   <property name="text">
    <string>Delete curve</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionOpen_one_curve">
   <property name="text">
    <string>Append curve from file</string>
   </property>
  </action>
  <action name="actionSave_current_curve">
   <property name="text">
    <string>Save current curve</string>
   </property>
  </action>
  <action name="actionSave_current_curve_as">
   <property name="text">
    <string>Save current curve as</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QWidget</extends>
   <header>qwt_plot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
